test/conftest.py

```diff
@@ -0,0 +1,40 @@
+# This file is used to augment the test configuration
+
+import os
+import pytest
+
+def pytest_addoption(parser):
+    parser.addoption("--volatility", action="store", default=None,
+        required=True,
+        help="path to the volatility script")
+
+    parser.addoption("--python", action="store", default="python3",
+        help="The name of the interpreter to use when running the volatility script")
+
+    parser.addoption("--image", action="append", default=[],
+        help="path to an image to test")
+
+    parser.addoption("--image-dir", action="append", default=[],
+        help="path to a directory containing images to test")
+
+def pytest_generate_tests(metafunc):
+    """Parameterize tests based on image names"""
+
+    images = metafunc.config.getoption('image')
+    for d in metafunc.config.getoption('image_dir'):
+        images = images + [os.path.join(d, x) for x in os.listdir(d)]
```

Comment:
Can we use a clearer variable name?

```suggestion
    for image_dir in metafunc.config.getoption('image_dir'):
        images = images + [os.path.join(image_dir, dir) for dir in os.listdir(image_dir)]
```